### ISSUE TITLE
place holder still shows ${ref}

### ISSUE THREAD

**Issue by hmy65** [LikeTheSalad/android-stem#33]
gradle 7.4
android gradle plugin 7.3.0

in the generated resolved.xml, it shows the correct string

but in app, it just shows ${ref} ...

any possible reasons? thanks.

**Comment by 108085** [comment]
got the same issue

**Comment by LikeTheSalad** [comment]
Hi! thanks for your feedback. Maybe Google changed the way to define extra resource dirs, I haven't taken a look at it yet, I've had a busy couple of days lately, though I'm planning to take a look this week

**Comment by Szczypiorek44** [comment]
Same issue here. Can't wait for the fix :)
android gradle plugin 7.3.0
gradle-7.4.2-all.zip

**Comment by francescofornasini** [comment]
Same issue, i resolved temporarily by adding these lines of code in the android block of the app's gradle
```
sourceSets {
        debug {
            res.srcDirs = ["${project.buildDir}/generated/resolved/debug/"]
        }
        release {
            res.srcDirs = ["${project.buildDir}/generated/resolved/release/"]
        }
    }
```

**Comment by hmy65** [comment]
> Same issue, i resolved temporarily by adding these lines of code in the android block of the app's gradle
> 
> ```
> sourceSets {
>         debug {
>             res.srcDirs = ["${project.buildDir}/generated/resolved/debug/"]
>         }
>         release {
>             res.srcDirs = ["${project.buildDir}/generated/resolved/release/"]
>         }
>     }
> ```

Confirmed.
By adding these lines of code under app:gradle android{} block, this problem has been temporarily fixed. 
Thanks for sharing.

**Comment by LikeTheSalad** [comment]
Thanks for the feedback guys, this should be fixed now on version `2.2.3` - Please feel free to reopen it if you still get the issue.
